tests/integration/test_system_containers_images.sh

```diff
@@ -106,29 +106,30 @@ ${ATOMIC} images list -f type=ostree > ${WORK_DIR}/images.out
 assert_matches "atomic-test-system" ${WORK_DIR}/images.out
 
 
+# This part got commented out because of the new skopeo copy way
```

Comment:
If these are no longer needed then go ahead and remove them. If they are still legit tests please reenable or replace them.

Reply:
sounds good, will do that when forcing push new commits. :)